components/Header/index.tsx

```diff
@@ -1,14 +1,78 @@
 "use client";
 
-import { Children, ReactNode, useEffect, useState } from "react";
+import { useEffect, useState } from "react";
+import Image from "next/image";
 import Search from "@/components/Search";
-import React from "react";
+import idsLogo from "../../public/assets/images/ids/IDS logo_landscape_WHITE.svg";
```

Comment:
Can we fix the file path for this to include the _ after IDS please? Just the inconsistency makes me twitch, I realize it's likely not a functional problem :)

Reply:
Peace of mind added